htdocs/js/apps/PGCodeMirror/pgeditor.js

```diff
@@ -69,7 +69,7 @@
 	};
 
 	const cm = webworkConfig.pgCodeMirror = CodeMirror.fromTextArea(document.querySelector('.codeMirrorEditor'), {
-		mode: 'PG',
+		mode: document.querySelector('.codeMirrorEditor').dataset.mode,
```

Comment:
This would be better as
```suggestion
		mode: document.querySelector('.codeMirrorEditor')?.dataset.mode ?? 'PG',
```
This way if the `.codeMirrorEditor` element does not have the `mode` dataset attribute set, then it will still work.